<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="screen03" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>760</width>
      <height>570</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="screen01" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="screen02" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>621</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>621</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="white">
    <property name="geometry">
     <rect>
      <x>708</x>
      <y>621</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>621</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yellow">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>621</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rednum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bluenum">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>730</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="greennum">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>730</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yellownum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>730</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="whitenum">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>730</y>
      <width>72</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>830</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR ALL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorpicker">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>730</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="redpicker">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>730</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bluepicker">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>780</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="greenpicker">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>780</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowpicker">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>830</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>YELLOW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="whitepicker">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>830</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>WHITE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>680</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Setting first color ball value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accurate">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>610</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Accuration:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
